### ISSUE TITLE
Daemon refuses to start if DNS cannot resolve lbry.io domain

### ISSUE THREAD

**Issue by lyoshenka** [lbryio/lbry#908]
The error message says it cannot connect to the internet, but the truth is that it cannot connect to lbry.io because the DNS resolve failed. Internet connectivity check needs to be more robust.

```
$ /opt/lbry/lbrynet-daemon --version
{
  "build": "release",
  "installation_id": "5GfsGqN89ysnaQiorZZNg3P3vDbc5WWJ75FbNDEiCoZXpzE4beznwMTHbNPYLww8gA",
  "lbrynet_version": "0.16.0",
  "lbryschema_version": "0.0.11",
  "lbryum_version": "3.1.7",
  "os_release": "4.4.0-93-generic",
  "os_system": "Linux",
  "platform": "Linux-4.4.0-93-generic-x86_64-with-Ubuntu-16.04-xenial",
  "processor": "x86_64",
  "python_version": "2.7.12"
}
$ /opt/lbry/lbrynet-daemon
2017-09-20 13:40:34,369 INFO     lbrynet.daemon.DaemonControl:70: Starting lbrynet-daemon from command line
2017-09-20 13:40:34,373 INFO     lbrynet.core.utils:108: Failed to connect to lbry.io:80. Maybe the internet connection is not working
Traceback (most recent call last):
  File "site-packages/lbrynet/core/utils.py", line 101, in check_connection
gaierror: [Errno -2] Name or service not known
2017-09-20 13:40:34,374 INFO     lbrynet.daemon.DaemonControl:77: Not connected to internet, unable to start
```

**Comment by rkrp** [comment]
I would like to take this up! Would appreciate any pointers.

**Comment by tzarebczan** [comment]
@rkrp looks like we already have a PR for this, but it wasn't merged yet.

**Comment by jackrobison** [comment]
Merged #912 [here](https://github.com/lbryio/lbry/commit/b9e03a7f134bcc1704052a525426da28550a92d3)
